src/aap_eda/core/models/credential_type.py

```diff
@@ -20,6 +20,8 @@
 
 
 class CredentialType(BaseOrgModel, UniqueNamedModel):
+    router_basename = "credentialtype"
```

Comment:
@bzwei Should this be a meta property?

Reply:
no. it is a class variable. I followed the example of DAB